example/service/ec2/filterInstnaces/filter_ec2_by_tag.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"log"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"strings"
+	"os"
+)
+
+var awsRegions = []string{"us-east-1", "us-west-1", "us-west-2", "eu-west-1", "eu-central-1", "sa-east-1"}
```

Comment:
Yea, that makes sense. We want our examples to be as simple as possible. That way when users who want to add their own customizations to them, can.